tests/integration-tests/tests/storage/test_fsx_lustre.py

```diff
@@ -71,6 +72,58 @@ def test_fsx_lustre(
     _test_data_repository_task(remote_command_executor, mount_dir, bucket_name, fsx_fs_id, region)
 
 
+@pytest.mark.regions(["us-east-1"])
+@pytest.mark.instances(["c5.xlarge", "m6g.xlarge"])
+@pytest.mark.schedulers(["sge"])
+@pytest.mark.usefixtures("os", "instance", "scheduler")
+# FSx is not supported on CentOS 6
+@pytest.mark.skip_oss(["centos6"])
+# FSx is only supported on ARM instances for Ubuntu 18.04 and Amazon Linux 2
+@pytest.mark.skip_dimensions("*", "m6g.xlarge", "alinux", "*")
+@pytest.mark.skip_dimensions("*", "m6g.xlarge", "centos7", "*")
+@pytest.mark.skip_dimensions("*", "m6g.xlarge", "ubuntu1604", "*")
+def test_fsx_lustre_backup(
+    region, pcluster_config_reader, clusters_factory, test_datadir, os,
+):
+    """
+    Test all FSx Lustre related features.
+
+    Grouped all tests in a single function so that cluster can be reused for all of them.
+    """
```

Comment:
Can you update this docstring? In it, can you summarize all of the functionality being tested (automatic backup time, backups not deleted, backup contains test file which is present when restoring  a file system from it)?